bndtools.core/src/bndtools/command/OpenProjectHandler.java

```diff
@@ -75,10 +79,25 @@ private static void selectProject(IProject project) throws Exception {
 				.getId());
 		}
 
+		IResource select = getNotNull(() -> project.findMember(Workspace.BUILDFILE),
```

Comment:
+1 Nice! I see this as more useful for Bndtools than simply selecting the project.